user-guide/Advanced_Functionality/Security/Advanced_security_configuration/Configuring_external_authentication_via_an_identity_provider_using_SAML.md

```diff
@@ -242,11 +242,12 @@ From DataMiner 10.2.0/10.1.12 onwards, users authenticated by Azure AD using SAM
      ...
    </DataMiner>
    ```
+1. User groups have to exist in DataMiner for users to be added to them (both for Groups claims set to true and false). See [Adding a user group] (Adding_a_user_group) for more info on how to do this
```

Comment:
The step to add user groups is already the second step of this procedure, which makes this a bit weird. When exactly should the user add the user groups? Possibly this should be rewritten as a note instead?
Also, make sure you leave an empty line above this line, and that you format cross-references correctly. In this case, it should be `[Adding a user group](xref:Adding_a_user_group)`.

Reply:
Moved this line to the second step to make the original more clear